src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -222,12 +222,31 @@ interface IOptionSettlementEngine {
         // These are 1:1 contracts with the underlying Option struct
         // The number of contracts written in this claim
         uint112 amountWritten;
-        // The amount of contracts assigned for exercise to this claim
-        uint112 amountExercised;
         // The two amounts above along with the option info, can be used to calculate the underlying assets
         bool claimed;
     }
 
+    /// @dev Claims are options lots which are able to have options added to them on different
+    /// bucketed days. This struct is used to keep track of how many options in a single lot are
+    /// written on each day, in order to correctly perform fair assignment.
+    struct ClaimIndex {
+        // The amount of options written on a given day
```

Comment:
General nit on struct variable docs — could move into @param natspec after the @dev natspec

Reply:
agree in general. ok w/u if we take this in a different fix-up PR? might make this one a bit noisier to include it here.

Reply:
Agreed, can do a separate PR for a general NatSpec pass